### PR TITLE
Fix documentation coherence, and wrong order of arguments

### PR DESCRIPTION
**Pull request by yohann-bacha**
As a convention, the app options should be written as its long form.
And since CLI version upgrade, the order of the arguments has changed.

### CODE REVIEW

File: src/_posts/platform/app/2000-01-01-scaling.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ data with our [CLI]({% post_url platform/cli/2000-01-01-start %}) with the
 following command:
 
 ```bash
-$ scalingo -a <appname> stats
+$ scalingo --app <appname> stats
```
Comment (EtienneM) [inline src/_posts/platform/app/2000-01-01-scaling.md:51]: nitpick: we use `my-app` as the application name throughout the doc if you could also update that here :)

```suggestion
$ scalingo --app my-app stats
```

File: src/_posts/platform/getting-started/2000-01-01-getting-started-with-metabase.md [hunk 2]
```diff
@@ -49,5 +49,5 @@ To update your Metabase application to the latest version, you need to redeploy
 Via this command for example:
 
 ```
-scalingo -a  my-app deploy https://github.com/Scalingo/metabase-scalingo/archive/refs/heads/master.tar.gz
+scalingo --app  my-app deploy https://github.com/Scalingo/metabase-scalingo/archive/refs/heads/master.tar.gz
```
Comment (EtienneM) [inline src/_posts/platform/getting-started/2000-01-01-getting-started-with-metabase.md:52]: nitpick: there is an extra space

```suggestion
scalingo --app my-app deploy https://github.com/Scalingo/metabase-scalingo/archive/refs/heads/master.tar.gz
```

**Review by EtienneM — COMMENTED** [review]
Just a couple of picky comments if you could also fix them. Then LGTM :)

**Comment by yohann-bacha** [comment]
@EtienneM followed your nitpicks!

**Comment by btrd** [comment]
@yohann-bacha @EtienneM Can this PR be merged?

**Pull request merged**